packages/playwright/src/common/configLoader.ts

```diff
@@ -111,6 +114,15 @@ export async function loadConfig(location: ConfigLocation, overrides?: ConfigCLI
       project.teardown = undefined;
     }
   }
+
+  // 3. Load transform options from the playwright config.
+  const babelPlugins = (userConfig as any)['@playwright/test']?.babelPlugins || [];
+  const external = userConfig.build?.external || [];
+  setTransformConfig({ babelPlugins, external });
+
+  // 4. Sync new transform options to ESM loader.
+  await configureESMLoader();
```

Comment:
can this be called just once (above) ?

Reply:
Or just a separate call to `await loaderChannel.send('setTransformConfig', { config: transformConfig() });` wrapped in a function?